src/TensorNetwork.jl

```diff
@@ -400,7 +400,11 @@ end
     elseif set === :hyper
         map(first, Iterators.filter(((_, v),) -> length(v) >= 3, tn.indexmap))
     else
-        throw(MethodError(inds, "unknown query: $set"))
+      throw(ArgumentError("unknown query: set=$(set). 
+        Possible options are: 
+        :all, :open :inner :hyper :parallelto
+        For Quantum also :physical :virtual
+        "))
```

Comment:
Some suggestions:
- `parallelto` is another kwarg, not a option for `set`
- For múltiples lines, better use `"""`

I would to have these options generated automatically, but I don't know how to do it yet so I'm ok with this PR.

```suggestion
      throw(ArgumentError("""
        Unknown query: set=$(set) 
        Possible options are:
          - :all (default)
          - :open
          - :inner
          - :hyper
        For `AbstractQuantum`, the following are also available:
          - :physical
          - :virtual
        """))
```